.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release
+
+on:
+  release:
+    types: published
+
+jobs:
+  plugin_dst:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+
+      - name: Set up Python 3.8
```

Comment:
Is this needed twice?

Reply:
No it's not! :sweat_smile:  Feel free to remove the duplicate!